package.json

```diff
@@ -35,5 +35,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "react-router-dom": "^6.3.0"
```

Comment:
it should be dependencies not a devDependencies because it is needed in deployment